andino_firmware/.clang-format

```diff
@@ -2,7 +2,7 @@
 # http://clang.llvm.org/docs/ClangFormatStyleOptions.html
 
 BasedOnStyle:  Google
-ColumnLimit: 120
```

Comment:
Just a personal opinion: I'd say that 120 is better for readability but it's ok if you want to change it. andino_base's script is set to 120 IIRC.

Reply:
Yeah, if possible I'd rather set it to 100 since 120 forces me to horizontally scroll on my 15” laptop :).